plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlConfig.java

```diff
@@ -81,4 +82,16 @@ public MySqlConfig setDriverUseInformationSchema(boolean driverUseInformationSch
         this.driverUseInformationSchema = driverUseInformationSchema;
         return this;
     }
+
+    public boolean isEnforceTimestampMillisPrecision()
+    {
+        return enforceTimestampMillisPrecision;
+    }
+
+    @Config("mysql.enforce-millis-timestamp-precision")
```

Comment:
`@ConfigDescription`

Reply:
add `@Deprecated` and `@ConfigHidden`, and `legacy` in the config name